### ISSUE TITLE
Complaining about 'pp' in a string

### ISSUE THREAD

**Issue by skateman** [ManageIQ/miq_bot#235]
Hi,

if there is `'pp'` in a string in an `.rb` file, the bot complains about debugging statements:
![screenshot from 2016-12-07 10-05-12](https://cloud.githubusercontent.com/assets/649130/20961627/0ddf7092-bc66-11e6-8015-a301dd8d4cdf.png)

![screenshot from 2016-12-07 10-15-23](https://cloud.githubusercontent.com/assets/649130/20961629/1535bedc-bc66-11e6-8820-7eeb60bbb6ab.png)

**Comment by Fryguy** [comment]
Yeah, I was wondering when this might happen...we implemented very naively to avoid having to parse Ruby...guess we'll have to parse Ruby :)
